File: Front/mutpio/src/app/Models/ContratResiliation.ts
import {Devis} from "./Devis";

export class ContratResiliation {
  idCompteRes!: number;
  nom!: string;
  prenom!: string;
  numContrat!: string;
  organisme!: string;
  etat!: boolean;
  devis!: Devis;
}

File: Back/src/main/java/com/mutpio/pfe/Entities/ContratResiliation.java
package com.mutpio.pfe.Entities;


import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.FieldDefaults;

import java.io.Serializable;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class ContratResiliation implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idCompteRes;

    String nom, prenom, numContrat, organisme;

    Boolean etat;

    @OneToOne
    Devis devis;

}
